packages/rocketchat-mentions/server/Mentions.js

```diff
@@ -2,7 +2,7 @@
 * Mentions is a named function that will process Mentions
 * @param {Object} message - The message object
 */
-import Mentions from './Mentions';
+import Mentions from '../client/Mentions';
```

Comment:
Is that right? Server importing client?

Reply:
yes! It extends the mentions class from the client (as it was happening in the previous behaviour) and because of Mentions.js was moved to the client folder I had to change this line